src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/Microsoft.Extensions.Telemetry.Abstractions.csproj

```diff
@@ -19,6 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\ToBeRemoved\Options.ValidateOnStart\Options.ValidateOnStart.csproj" />
```

Comment:
I was under the impression that we were able to consume the .NET 8 version of this, is that not the case?

Reply:
Good point

Reply:
Not yet, it's the last bit of ToBeRemoved functionality left. I think the PR went into dotnet/runtime yesterday or this morning, so this should be removed soon....